opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -2230,7 +2230,7 @@ namespace Opm
         WellGroupHelpers::FractionCalculator fcalc(schedule, summaryState, well_state, current_step_, guide_rate_, tcalc.guideTargetMode(), pu, false, injectionPhase);
 
         auto localFraction = [&](const std::string& child) {
-            return fcalc.localFraction(child, "");
+            return fcalc.localFraction(child, child);
         };
```

Comment:
This change shouldnt be needed. Child is already on grup control and you don't need to pass its name here. ACTIONX_M1 runs perfectly without it.

Reply:
Ehhh - no that I do not agree. It runs through also in master, but the problem is that injection in well `WI1` starts to late. The ACTIONX triggers at 2020-07-20 and injection should start immediately after that, if I remove `(child,child)` change injection will not start before the next report step - at 2020-08-01.

Reply:
Removing this did not change anything for me. Results, number of iterations etc. are the same.

Reply:
You should run flow with `--solver-max-time-step-in-days=1` to ensure that the ACTIONX triggers well before the report step ends.